src/AdornedRulerPanel.cpp

```diff
@@ -1065,7 +1065,6 @@ std::vector<UIHandlePtr> AdornedRulerPanel::QPCell::HitTest(
    std::vector<UIHandlePtr> results;
    auto xx = state.state.m_x;
 
-#ifdef EXPERIMENTAL_DRAGGABLE_PLAY_HEAD
```

Comment:
I'm not sure this function does anything at the moment, it seems to me like instead of dragging the playhead around, I instead do things with loop regions.

Reply:
Can’t you click precisely on the down pointing green (or red) triangle and move it, when the head is in pinned state?

Reply:
I implemented the pinned head, which at first was always centered, then in a later release let you drag it off center, thinking that it’s more useful during recording to have it farther right.

Also little known perhaps — you can double click it to recenter.

Reply:
oh, in pinned state. Yes, that still works.

Reply:
Good